src/Microsoft.OpenApi.Readers/ParseNodes/OpenApiAnyConverter.cs

```diff
@@ -195,16 +196,23 @@ public static IOpenApiAny GetSpecificOpenApiAny(IOpenApiAny openApiAny, OpenApiS
 
             if (type == "string" && format == "byte")
             {
-                if (byte.TryParse(value, NumberStyles.Any, CultureInfo.InvariantCulture, out var byteValue))
+                try
                 {
-                    return new OpenApiByte(byteValue);
+                    return new OpenApiByte(Convert.FromBase64String(value));
                 }
+                catch(Exception)
+                { }
             }
 
             // binary
             if (type == "string" && format == "binary")
             {
-                return new OpenApiBinary(Convert.FromBase64String(value));
+                try
+                {
+                    return new OpenApiBinary(Encoding.UTF8.GetBytes(value));
+                }
+                catch(Exception)
+                { }
```

Comment:
Try/catch unnecessary.

Reply:
Leave the uplevel to process the exception?

Reply:
Which exception case are we envisioning here?

---
In reply to: [265768186](https://github.com/Microsoft/OpenAPI.NET/pull/391#discussion_r265768186) [](ancestors = 265768186)

Reply:
GetBytes(..) may throw the following exception:

Exceptions:
        //   T:System.ArgumentNullException:
        //     s is null.
        //
        //   T:System.Text.EncoderFallbackException:
        //     A fallback occurred (see Character Encoding in the .NET Framework for complete
        //     explanation)-and-System.Text.Encoding.EncoderFallback is set to System.Text.EncoderExceptionFallback.

Reply:
Got it. Let's do this:
- The ArgNullEx one won't be thrown since we already handle it above.
- Can we specifically catch the second type of exception? 

---
In reply to: [265771477](https://github.com/Microsoft/OpenAPI.NET/pull/391#discussion_r265771477) [](ancestors = 265771477)

Reply:
Changed.